data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -67,11 +67,18 @@
 
   protected final JsonFactory _factory;
 
+  private boolean _sortProperties;
```

Comment:
Should've commenting on this earlier, but this should be renamed to `_sortKeys`, since "properties" aren't a concept that's inherently known by the JSON codec.

Also, just making sure... this doesn't sort array values as well, right?

Reply:
Good catch. It should be `_sortKeys` and it doesn't sort arrays. Let me make sure I check that in the unit test.